sycl/include/sycl/ext/intel/esimd/detail/simd_obj_impl.hpp

```diff
@@ -359,10 +359,6 @@ class simd_obj_impl {
   /// with l-value contexts in inline assembly.
   raw_vector_type &data_ref() { return M_data; }
 
-  /// Commit the current stored underlying raw vector to memory.
-  /// This is required when using inline assembly with private global variables.
-  void commit() { __esimd_vstore<RawTy, N>(&M_data, M_data); }
```

Comment:
Per our PM we can remove this method this time.
We could though have a "deprecated function" message and make it NOP: void commit() {}

I see that ABI breaking window is already open (April 11 to July), removing this method without deprecation may be an exception this time, but to be more polite, having a deprecated message for month or two would be safer/better, IMO

Reply:
yeah i dont have strong opinion, ill do this, thx